tests/autoyast/installation.pm

```diff
@@ -345,6 +351,7 @@ sub run {
     push(@needles, 'opensuse-welcome') if opensuse_welcome_applicable;
     # There will be another reboot for IPMI backend
     push @needles, qw(prague-pxe-menu qa-net-selection) if check_var('BACKEND', 'ipmi');
+
```

Comment:
Could you please remove this empty line from the commit.

Reply:
Done